Prac_10/testing.py

```diff
@@ -0,0 +1,95 @@
+"""
+CP1404/CP5632 Practical
+Testing demo using assert and doctest
+"""
+
+import doctest
+from Prac_06.car import Car
+
+
+def repeat_string(s, n):
+    """Repeat string s, n times, with spaces in between."""
+    # return s * n
+    """The original function did not print with spaces in between each string?!"""

```

Comment:
No, because it was broken. You needed to fix it. The tests show what was supposed to happen.
Changing correct but failing tests is just pretending everything is OK when it's not.